NewSpring/ApollosProject
# App Clean Up

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR:

- Customized the Prayer Preview Cards and updated styles
- Updated Add Prayer Card styles
- Updated Prayer Menu styles
- Updated the Connect page so that all menu items point to the correct place. Note: the "Open Testing Panel" should no longer show in production.
- Updated all `@apollosproject` packages to `0.8.3` (which had the added bonus of fixing the warnings on the Add Prayer Form.
- Updated User Prayer Card styles.
- Disabled swiping on the tab view until we can figure out how to get the tabs and the menu to move in sync.
- 

### How do I test this PR?
Run through the prayer module and make note of any weird styling issues you might see.
Make sure that the items on the Connect tab go to the correct places.

---
> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [n/a] Closes [DEV-XXX](#url)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
